guide/packages.md

```diff
@@ -0,0 +1,155 @@
+| membrane_scissors_plugin | Element for cutting off parts of the stream | [Hex](https://hex.pm/packages/membrane_scissors_plugin)&nbsp;[Docs](https://hexdocs.pm/membrane_scissors_plugin)&nbsp;[GitHub](https://github.com/membraneframework/membrane_scissors_plugin) |
+| membrane_element_tee     |                                             | [Hex](https://hex.pm/packages/membrane-element-tee)&nbsp;[Docs](https://hexdocs.pm/membrane-element-tee)&nbsp;[GitHub](https://github.com/membraneframework/membrane-element-tee)             |
+| membrane_element_pcap    | [Experimental]                              | [GitHub](https://github.com/membraneframework/membrane-element-pcap)                                                                                                                          |
+| membrane_ice_plugin      | [In progress]                               | [GitHub](https://github.com/membraneframework/membrane_ice_plugin)                                                                                                                            |
+
+### Media network protocols & containers
+
+| Package                              | Description                                                                                      | Links                                                                                                                                                                                                                             |
+| ------------------------------------ | ------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| membrane_rtp_plugin                  | [Alpha] Membrane bins and elements for handling RTP and RTCP streams                             | [Hex](https://hex.pm/packages/membrane_rtp_plugin)&nbsp;[Docs](https://hexdocs.pm/membrane_rtp_plugin)&nbsp;[GitHub](https://github.com/membraneframework/membrane_rtp_plugin)                                                    |
+| membrane_bin_rtp                     | Bin consuming RTP stream. Will be deprecated when membrane_rtp_plugin goes stable.               | [Hex](https://hex.pm/packages/membrane-bin-rtp)&nbsp;[Docs](https://hexdocs.pm/membrane-bin-rtp)&nbsp;[GitHub](https://github.com/membraneframework/membrane-bin-rtp)                                                             |
+| membrane_element_rtp                 | Elements for consuming RTP stream. Will be deprecated when membrane_rtp_plugin goes stable.      | [Hex](https://hex.pm/packages/membrane-element-rtp)&nbsp;[Docs](https://hexdocs.pm/membrane-element-rtp)&nbsp;[GitHub](https://github.com/membraneframework/membrane-element-rtp)                                                 |
```

Comment:
Since JitterBuffer has already been deprecated these should be too. They are included in rtp_plugin IIRC